src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeOptionsFormDataProvider.php

```diff
@@ -60,6 +71,33 @@ public function getData()
      */
     public function setData(array $data)
     {
+        if ($errors = $this->validate($data)) {
```

Comment:
In this case we all understand that is an affectation and not a comparison but best practices try to avoid this.
Neither CSFixer and Stan didn't pop a warning ?

Reply:
I don't thinke we have such rules in cs or stan configs, but we could indeed clean this up